devops_toolset/core/log_tools.py

```diff
@@ -38,7 +38,7 @@ def log_stdouterr(output: bytes, level: LogLevel):
     """
 
     for line in output.splitlines():
-        logging.log(level.value[0], line.decode("utf-8"))
+        logging.log(level.value[0], line.decode("cp1252"))
```

Comment:
Is it not working with UTF-8?

Reply:
Some logs from the dotnet tool raised an encoding error while returning from call_subprocess. There is a workaround to maintain the utf-8 encoding and ignore possible errors while handling unsupported characters:

`logging.log(level.value[0], line.decode(encoding="utf-8", errors="ignore"))`

Maybe we can try this one.